### DISCUSSION TITLE
nextcloud talk only works on the local network

### DISCUSSION THREAD

**Discussion by Spp595** [nextcloud/all-in-one#3884] [Questions] [answered]
Nextcloud AIO v7.7.1
Nginx proxy manager

I installed nextcloud aio using the reverse proxy nginx proxy manager. I configured everything according to the documentation, but talk does not work. If 2 devices are on the home local network, then talk works. If one of the devices is via the internet, it will not be able to connect and will reconnect

![изображение](https://github.com/nextcloud/all-in-one/assets/104263581/8f69fb6b-0356-4f75-a0bb-747abb8b3533)
![изображение](https://github.com/nextcloud/all-in-one/assets/104263581/9b41d55b-e850-4f4b-9304-6467c1326a7f)
![изображение](https://github.com/nextcloud/all-in-one/assets/104263581/be7fd742-f3d6-4c8c-a827-76a0e7f36faa)

everything else, even including office which did not work for me in the docker installation works well

also in: Checking security and settings
Your remote address was identified as "192.168.16.1" and is currently blocked by brute force protection, although on my local network IP addresses like 10.10.10.1

**Comment by szaimen** [comment]
Hi, did you open port 3478/tcp and 3478/udp in your firewall?

**Comment by Spp595** [comment]
Yes

**Comment by szaimen** [comment, marked as answer]
Then can you follow https://github.com/nextcloud/all-in-one/discussions/1358?

**Comment by szaimen** [comment]
Moving to discussions since I cannot reproduce the issue on my test instance
